src/mpid/ch4/netmod/ofi/ofi_progress.c

```diff
@@ -68,9 +68,7 @@ int MPIDI_OFI_progress(int vci, int blocking)
         return MPI_SUCCESS;
     }
 
-    if (unlikely(MPIDI_OFI_get_buffered(wc, 1)))
-        mpi_errno = MPIDI_OFI_handle_cq_entries(wc, 1);
-    else if (likely(1)) {
+    {
```

Comment:
Why keep this?

Reply:
Just not to add the indentation noise for the review.

Reply:
Personally I'd rather just type `--ignore-all-space` or click the checkmark in GitHub and not have the weirdly formatted code forever.

Reply:
> click the checkmark in GitHub and not have the weirdly formatted code forever.

Where is the (which) checkmark?

Reply:
<img width="669" alt="Screen Shot 2020-10-16 at 9 10 24 AM" src="https://user-images.githubusercontent.com/1561207/96269081-7dc90c80-0f8f-11eb-829d-14ee7645da9d.png">

Reply:
Ahh, thanks! Never noticed it was there :smiley: